cloud: billing cost explorer (#16608)
pingcap/docs: tidb-cloud/tidb-cloud-billing.md

> - The total amount in the monthly bill is rounded off to the 2nd decimal place.
> - The total amount in the daily usage details is accurate to the 6th decimal place.

> 

Reviewer: https://docs.pingcap.com/tidbcloud/manage-user-access#organization-role

Do we also need to add the info about viewing usage costs here?

![image](https://github.com/pingcap/docs/assets/79440533/0e7e0143-264e-4516-ae27-fbccbafb5b0b)

Author: https://github.com/pingcap/docs/pull/16608/commits/2f0b028936c729833deada833e7e874bc464d962

## Cost explorer

If you are in the `Organization Owner` or `Organization Billing Admin` role of your organization, you can view and analyze the usage costs of TiDB Cloud. Otherwise, skip this section.

To analyze and customize your cost reports of your organization, perform the following steps:

> 1. Click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner of the TiDB Cloud console.

Reviewer: ```suggestion
1. In the lower-left corner of the [TiDB Cloud console](https://tidbcloud.com), click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.
```

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

> 2. In the left navigation pane, navigate to **Billing > Cost Explorer**.

Reviewer: 这里是不是要先在左下角点 Billing
![image](https://github.com/pingcap/docs/assets/79440533/c25786f1-0e9e-4cf7-81d3-45aff17dd296)

> 2. In the left navigation pane, navigate to **Billing > Cost Explorer**.

Author: ```suggestion
2. In the left navigation pane, click the **Billing** tab and then click the **Cost Explorer** tab.
```

> 2. In the left navigation pane, click the **Billing** tab and then click the **Cost Explorer** tab.

Reviewer: ```suggestion
2. On the **Billing** page, click the **Cost Explorer** tab.
```

3. On the **Cost Explorer** page, expand the **Filter** section in the upper-right corner to customize your report. You can set the time range, select a grouping option (such as by service, project, cluster, region, product type, and charge type), and apply filters by selecting specific services, projects, clusters, or regions. The cost explorer will display you with the following information:

    - **Cost Graph**: visualizes the cost trends over the selected time range. You can switch between **Monthly**, **Daily**, and **Total** views.

>     - **Cost Breakdown**: displays a detailed breakdown of your costs according to the chosen grouping option. For further analysis, you can download the data in CSV format.

Reviewer: ```suggestion
    - **Cost Breakdown**: displays a detailed breakdown of your costs according to the selected grouping option. For further analysis, you can download the data in CSV format.
```
